### ISSUE TITLE
NVDA does not read the list of recent files in the one drive system tray icon

### ISSUE THREAD

**Issue by fatihyuksek1** [nvaccess/nvda#12382]
### Steps to reproduce:
1. Add or edit one or more file in one drive
2. Press enter on the one drive system tray icon
3. Press up and down arrow keys to navigate the list of recent files
4. You will notice that NVDA does not read anything in the list
5. Turn on narrator
6. Repeat the steps above with narrator
7. You will notice tht narrator will read the list as you go up and down within the list
### Actual behavior:
NVDA will not read the list when up or down arrow is pressed.
### Expected behavior:
NVDA reads the ites in the list of recent files, in the system tray on one drive
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.4
#### Windows version:
windows 10 1909
#### Name and version of other software in use when reproducing the issue:
One drive
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes. They were the same as 2020.4
#### If add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
didn't try

**Comment by Adriani90** [comment]
@fatihyuksek1 are you still having this issue with NVDA 2024.4.1?

**Comment by fatihyuksek1** [comment]
I’m not sure as I don’t use one drive anymore.Adriani90 ***@***.***> şunları yazdı (24 Kas 2024 16:57):﻿
@fatihyuksek1 are you still having this issue with NVDA 2024.4.1?

**Comment by Adriani90** [comment]
Tested with NVDA 2024.4.1 and Onedrive Build 24.211.1020.0001 (64-Bit), NVDA reads the list of recent files as expected. Closing as works for me. if someone still has this issue, please comment and we can reopen.
